tests/components/LoginForm.spec.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import LoginForm from 'components/LoginForm';
+import { fireEvent, wait } from '@testing-library/react-native';
+
+import { renderWithRedux, configureStore } from '../helpers';
+
+describe('<LoginForm />', () => {
+  let wrapper;
+  let store;
+  const props = {
+    onSubmit: jest.fn(),
+  };
+
+  beforeEach(() => {
+    store = configureStore();
+    wrapper = renderWithRedux(<LoginForm {...props} />, store);
+  });
+
+  describe('Email Input', () => {
+    let input;
+    beforeEach(() => {
+      input = wrapper.queryByTestId('email-input');
+    });
+
+    it('should display an email field', () => {
+      expect(input).toBeTruthy();
+    });
+
+    describe('when the email input is not valid', () => {
+      beforeEach(() => {
+        fireEvent.changeText(input, 'example');
+        fireEvent.blur(input);
+      });
+
+      it('should show a email is not valid error', () => {
+        expect(wrapper.queryAllByLabelText('form-error')).toHaveLength(1);
+        expect(wrapper.queryByText('Email is not a valid email')).toBeTruthy();
```

Comment:
what do you think about defining the errors as an object or constants at the top of the file so you are not repeating the hardcoded versions throughout the file?